.travis.yml

```diff
@@ -1,14 +1,31 @@
 language: swift
 osx_image: xcode10.2
-env:
-  - SKIP_RPC_TESTS=1
 
 xcode_workspace: CKB.xcworkspace
 xcode_scheme: CKB
 xcode_destination: platform=OS X,arch=x86_64
+
 before_install:
   - gem install cocoapods -N
   - pod --version
   - travis_wait pod repo update --silent
+
+matrix:
+  include:
+    - name: Tests without RPC
+      if: NOT branch = master
```

Comment:
Not test for master branch ?

Reply:
See the below test that for other master.